src/character.h

```diff
@@ -12,10 +12,10 @@
 
 using skill_id = string_id<Skill>;
 enum field_id : int;
-class field;
-class field_entry;
+    class field;
```

Comment:
You have to look out for this one, there's a bug in astyle that will indent improperly after an enum declaration with a size specified.
AFAIK this means there are certain files where we won't be able to add them to the whitelist until there is a fix for astyle or we find a workaround.